src/helpers/generator.test.ts

```diff
@@ -527,5 +527,134 @@ describe('generator', () => {
+                signals: [
+                  {
+                    name: 'navigation',
+                    url: 'https://www.google.com/search?q=hello+world&source=hp&ei=Wa-wYt7xDPa15NoP6NeGmAk&iflsig=AJiK0e8AAAAAYrC9aQ-K5PLjczC0c7dceKI--pjoitdO&ved=0ahUKEwieguz8x7z4AhX2GlkFHeirAZMQ4dUDCAo&uact=5&oq=hello+world&gs_lcp=Cgdnd3Mtd2l6EAMyCAgAEIAEELEDMgsILhCABBCxAxDUAjIICAAQgAQQsQMyCAgAEIAEELEDMggIABCABBCxAzIFCC4QgAQyCAgAEIAEELEDMggIABCABBCxAzILCC4QgAQQsQMQ1AIyBQgAEIAEOg4IABCPARDqAhCMAxDlAjoOCC4QjwEQ6gIQjAMQ5QI6EQguEI8BENQCEOoCEIwDEOUCOhEILhCABBCxAxCDARDHARCjAjoLCC4QgAQQxwEQrwE6CAgAELEDEIMBOgQIABADOgsIABCABBCxAxCDAToICC4QsQMQgwE6DgguEIAEELEDEMcBEKMCOhEILhCABBCxAxCDARDHARCvAToOCC4QgAQQxwEQrwEQ1AI6DgguELEDEIMBEMcBEKMCOhEILhCABBCxAxCDARDHARDRAzoLCC4QgAQQsQMQgwE6CAguEIAEELEDOg4IABCABBCxAxCDARDJAzoQCC4QgAQQsQMQxwEQ0QMQCjoICAAQgAQQyQM6BwgAELEDEApQrwJYvAtgsgxoAXAAeACAAVeIAbgFkgECMTGYAQCgAQGwAQo&sclient=gws-wiz',
+                  },
+                  {
+                    name: 'navigation',
+                    url: 'https://www.google.com/search?q=hello+world&source=hp&ei=Wa-wYt7xDPa15NoP6NeGmAk&iflsig=AJiK0e8AAAAAYrC9aQ-K5PLjczC0c7dceKI--pjoitdO&ved=0ahUKEwieguz8x7z4AhX2GlkFHeirAZMQ4dUDCAo&uact=5&oq=hello+world&gs_lcp=Cgdnd3Mtd2l6EAMyCAgAEIAEELEDMgsILhCABBCxAxDUAjIICAAQgAQQsQMyCAgAEIAEELEDMggIABCABBCxAzIFCC4QgAQyCAgAEIAEELEDMggIABCABBCxAzILCC4QgAQQsQMQ1AIyBQgAEIAEOg4IABCPARDqAhCMAxDlAjoOCC4QjwEQ6gIQjAMQ5QI6EQguEI8BENQCEOoCEIwDEOUCOhEILhCABBCxAxCDARDHARCjAjoLCC4QgAQQxwEQrwE6CAgAELEDEIMBOgQIABADOgsIABCABBCxAxCDAToICC4QsQMQgwE6DgguEIAEELEDEMcBEKMCOhEILhCABBCxAxCDARDHARCvAToOCC4QgAQQxwEQrwEQ1AI6DgguELEDEIMBEMcBEKMCOhEILhCABBCxAxCDARDHARDRAzoLCC4QgAQQsQMQgwE6CAguEIAEELEDOg4IABCABBCxAxCDARDJAzoQCC4QgAQQsQMQxwEQ0QMQCjoICAAQgAQQyQM6BwgAELEDEApQrwJYvAtgsgxoAXAAeACAAVeIAbgFkgECMTGYAQCgAQGwAQo&sclient=gws-wiz',
+                    isAsync: true,
+                  },
+                ],
+                key: 'Enter',
+                modifiers: 0,
+              },
+              committed: true,
+              title: 'Press Enter',
+            },
+            {
+              pageAlias: 'page',
+              isMainFrame: true,
+              frameUrl:
+                'https://www.google.com/search?q=hello+world&source=hp&ei=Wa-wYt7xDPa15NoP6NeGmAk&iflsig=AJiK0e8AAAAAYrC9aQ-K5PLjczC0c7dceKI--pjoitdO&ved=0ahUKEwieguz8x7z4AhX2GlkFHeirAZMQ4dUDCAo&uact=5&oq=hello+world&gs_lcp=Cgdnd3Mtd2l6EAMyCAgAEIAEELEDMgsILhCABBCxAxDUAjIICAAQgAQQsQMyCAgAEIAEELEDMggIABCABBCxAzIFCC4QgAQyCAgAEIAEELEDMggIABCABBCxAzILCC4QgAQQsQMQ1AIyBQgAEIAEOg4IABCPARDqAhCMAxDlAjoOCC4QjwEQ6gIQjAMQ5QI6EQguEI8BENQCEOoCEIwDEOUCOhEILhCABBCxAxCDARDHARCjAjoLCC4QgAQQxwEQrwE6CAgAELEDEIMBOgQIABADOgsIABCABBCxAxCDAToICC4QsQMQgwE6DgguEIAEELEDEMcBEKMCOhEILhCABBCxAxCDARDHARCvAToOCC4QgAQQxwEQrwEQ1AI6DgguELEDEIMBEMcBEKMCOhEILhCABBCxAxCDARDHARDRAzoLCC4QgAQQsQMQgwE6CAguEIAEELEDOg4IABCABBCxAxCDARDJAzoQCC4QgAQQsQMQxwEQ0QMQCjoICAAQgAQQyQM6BwgAELEDEApQrwJYvAtgsgxoAXAAeACAAVeIAbgFkgECMTGYAQCgAQGwAQo&sclient=gws-wiz',
```

Comment:
can we remove these long URLs and just use dummy ones?

Reply:
yes, that's no problem. These URLs have no material impact and it will be much cleaner. In fact, we have the test object generators, I should have used them in the first place. I'll push a commit to accomplish this.